src/pkg/util/auth/token.go

```diff
@@ -0,0 +1,48 @@
+/*
+  Copyright (c) 2018, Sylabs, Inc. All rights reserved.
+
+  This software is licensed under a 3-clause BSD license.  Please
+  consult LICENSE file distributed with the sources of this project regarding
+  your rights to use or distribute this software.
+*/
+
+package auth
+
+import (
+	"io/ioutil"
+	"strings"
+)
+
+var (
+	// WarningTokenTooShort Warning return for token shorter than 200 b
+	WarningTokenTooShort = "Token is too short to be valid. Only pulls of public images will succeed"
```

Comment:
Not sure why it's useful to have these defined here - and if there is a reason they should be const.